client/src/main/java/io/confluent/kafka/schemaregistry/AbstractSchemaProvider.java

```diff
@@ -70,7 +68,7 @@ private void resolveReferences(Schema schema, Map<String, String> schemas, Set<S
       }
       if (!schemas.containsKey(reference.getName())) {
         String refSubject = QualifiedSubject.qualifySubjectWithParent(
-            DEFAULT_TENANT, schema.getSubject(), reference.getSubject());
+            schemaVersionFetcher().tenant(), schema.getSubject(), reference.getSubject());
```

Comment:
When exporting, would this be the tenant of the destination cluster?

Reply:
Yes